src/api/controllers/schemaController.ts

```diff
@@ -20,6 +20,13 @@ export async function readById(req: Request, res: Response) {
 }
 
 export async function create(req: Request, res: Response) {
+  const { slug, fields } = req.body
+  if (typeof slug !== "string" || !Array.isArray(fields))
+    throw new Error("Invalid schema format")
```

Comment:
Does this cause a client error (status 4XX)? Or a server error (5XX)? We should make sure it's the former.